scripts/rabbitmq-defaults

```diff
@@ -41,4 +41,20 @@ ENABLED_PLUGINS_FILE=${SYS_PREFIX}/etc/rabbitmq/enabled_plugins
 
 PLUGINS_DIR="${RABBITMQ_HOME}/plugins"
 
+# Search for another `plugins` directory towards the filesystem
+# root. This is useful when we want to have some common place to put
+# pre-packaged plugins. And $RABBITMQ_HOME usually contains RabbitMQ
+# version somewhere in the middle, making plugin packaging very
+# inconvinient.
+plugins_test_path="$(dirname $(dirname "$PLUGINS_DIR"))"
+while test "$plugins_test_path" != "/"; do
```

Comment:
If there are multiple rabbitmq versions installed, plugins from both will be added to this variable. 
Should there be just predefined "common plugins directory" e.g /usr/lib/rabbitmq/plugins instead?